bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -14,6 +14,10 @@ const intlMessages = defineMessages({
     id: 'app.presentation.presentationToolbar.nextSlideLabel',
     description: 'Next slide button label',
   },
+  slideOption: {
```

Comment:
We already have `previousSlideLabel`,  `nextSlideLabel` above.
`skipSlideLabel` ?

Reply:
In the rest of BBB it's referred to as GoToSlide